staff/sergio-ocaña/objects/bubblesort.js

```diff
@@ -0,0 +1,26 @@
+function sortObject(object) {
+    var elementsMoved = true
+    for (var i = 0; elementsMoved && (i < object.length); i++) {
+        elementsMoved = false
+        for (var j = 0; j < object.length - 1 - i; j++) {
+            var temp = 0
+            if (object[j] > object[j + 1]) {
+                temp = object[j + 1]
+                object[j + 1] = object[j]
+                object[j] = temp
+                elementsMoved = true
+            }
+        }
+    }
+}
+
```

Comment:
You could improve your code by using a while loop and just use a for to save lines, like I have.